sql/prepare/142_point_places.sql

```diff
@@ -70,14 +70,17 @@ create index on geocoder_point (osm_id);
 
 
 -- 5. Limit these polygons to a buffer radius dependent on the "place" tag.
-with place_sizes as (
+-- It was CTE but due to pg12 CTE materialized behavior and backward uncompatible
+-- materialized statemnt we rewrote in to temporary table
+create temporary table  place_sizes as (
```

Comment:
Extra space. Also, here I don't use brackets and indentation for `create table`, see line 13 for an example. But that's up to you, it looks good as it is.

Reply:
Ok. I reformat this code to make it in the same style